mistralrs-pyo3/Cargo.toml

```diff
@@ -17,7 +17,7 @@ doc = false
 
 [dependencies]
 pyo3.workspace = true
-mistralrs-core = { version = "0.1.7", path = "../mistralrs-core" }
+mistralrs-core = { version = "0.1.7", path = "../mistralrs-core", features = ["pyo3_macros"] }
```

Comment:
Can you please also update `mistralrs-pyo3/Cargo_template.toml`?

Reply:
Yeah forgot to add it there, is the `Cargo_template.toml` only used to select features for the `pypi` uploads?

Reply:
Yeah, it's used for creating packages with different features. I don't love that way of doing it, though, because we have so many packages uploaded. Do you know of another way to do this? Ideally, we could gate features on install, but maturin is missing functionality to do that.

Reply:
Well it seams to be possible to set feature flags via the `MATURIN_PEP517_ARGS` environment, as mentioned [here](https://github.com/PyO3/maturin/discussions/1090). But that is very unintuitive. 

Maybe we should create a ticket to enable features via pips optional dependency flags e.g. mistralrs[cuda] to use the `cuda` feature.

Reply:
For now i just added `pyo3_macros` as a forced feature. Should be good to go 👍